readoutcard.sh

```diff
@@ -32,6 +33,7 @@ cmake $SOURCEDIR                                                      \
       -DCMAKE_INSTALL_PREFIX=$INSTALLROOT                             \
       ${BOOST_VERSION:+-DBOOST_ROOT=$BOOST_ROOT}                      \
       ${COMMON_O2_VERSION:+-DCommon_ROOT=$COMMON_O2_ROOT}             \
+      ${CONFIGURATION_VERSION:+-DConfiguration_ROOT=$CONFIGURATION_ROOT} \
```

Comment:
Can configuration be optional or coming from the system? If not I would simply drop the `${CONFIGURATION_VERSION:+` part.